frontend/src/components/UI/TheNavigationDrawer.vue

```diff
@@ -72,3 +72,9 @@ export default defineComponent({
   },
 });
 </script>
+
+<style>
+.bg-color {
+  background-color: #03DAC6;
```

Comment:
It looks great!

This is just to remind us to update the hardcoded color once we get themes set up again. But we should merge in the meantime!